soc/arm/nordic_nrf/nrf53/Kconfig.defconfig.nrf5340_CPUAPP_QKAA

```diff
@@ -11,4 +11,17 @@ config SOC
 config NUM_IRQS
 	default 69
 
+config NRF5340_CPUAPP_ERRATUM19
+	bool "Include workarounds for Erratum 19"
+	default y
+	help
+	  This anomaly applies to IC Rev. Engineering A, build codes QKAA-AB0.
+	  This config MUST be enabled if there is a chance the code will be run
+	  on nRF5340 Engineering A. Enabling this config is safe on other
+	  nRF5340 variants, but might increase flash size.
```

Comment:
I'd say "it is not recommended as it increases flash size"

Reply:
I'm unsure. The recommendation is to keep the config on if there is a chance the code might be run on EngA chips, i.e. it's compiled once and possibly flashed to different variants. Leaving the wording as is lets them make the decision for themselves.